packages/@react-spectrum/dialog/docs/AlertDialog.mdx

```diff
@@ -0,0 +1,176 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/dialog';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {ActionButton} from '@react-spectrum/button';
+import {AlertDialog, DialogTrigger} from '@react-spectrum/dialog';
+```
+
+# AlertDialog
+
+<p>{docs.exports.AlertDialog.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['AlertDialog']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/dialog/#Options'}
+  ]} />
+
+## Example
+
+```tsx example
+<DialogTrigger>
+  <ActionButton>Save</ActionButton>
+  <AlertDialog title="Low Disk Space" variant="warning" primaryLabel="Confirm">
+    You are running low on disk space. Delete unnecessary files to free up space.
+  </AlertDialog>
+</DialogTrigger>
+```
+
+## Content
+
+Unlike Dialog, `slots` are not supported in AlertDialog since its layout is very deliberate.
+The body of an AlertDialog can be provided by passing children to the component. The AlertDialog also 
+supports three optional buttons in its footer: a confirmation button, a secondary button, and a cancel button.
+Each button can be rendered by providing a string to the `primaryLabel`, `secondaryLabel`, and `cancelLabel` 
+respectively.
+
+```tsx example
+<DialogTrigger>
+  <ActionButton>Exit</ActionButton>
+  <AlertDialog variant="information" title="Register profile" primaryLabel="Register" secondaryLabel="Remind me later" cancelLabel="Cancel">
+    You have not saved your profile information for this account. Would you like to register now?
+  </AlertDialog>
+</DialogTrigger>
+```
+
+### Accessibility
+
+If any of the buttons in the ActionDialog's footer should be focused on render, specify which one via the `autoFocusButton` prop.
+
+```tsx example
+<DialogTrigger>
+  <ActionButton>Save</ActionButton>
+  <AlertDialog variant="confirmation" title="Save file" primaryLabel="Save" cancelLabel="Cancel" autoFocusButton="primary">
+    A file with the same name already exists. Overwrite?
+  </AlertDialog>
+</DialogTrigger>
+```
+
+### Internationalization
+
+Be sure to localize any strings provided to the AlertDialog and to the button labels.  
```

Comment:
Feels kinda weird to have a bunch of single sentence sections, perhaps I should combine these subsections?

Reply:
Sure. That would be ok.